attributes/attributes.go

```diff
@@ -19,7 +19,10 @@
 // Package attributes defines a generic key/value store used in various gRPC
 // components.
 //
-// All APIs in this package are EXPERIMENTAL.
+// Experimental
+//
+// Notice: This package is experimental and may be changed or removed in a
```

Comment:
Nit: In all places where we say "This package/type/API is experimental", could you please make the experimental all caps. Thank you. 

Example: https://github.com/grpc/grpc-go/blob/master/credentials/xds/xds.go#L24